tests/test.js

```diff
@@ -0,0 +1,42 @@
+const assert = require('assert')
+const spawn = require('child_process').spawn
+const uuidV4 = require("uuid/v4")
+const fs = require('fs')
+
+describe('WebUi generator', () => {
+    
+    let uniqueId = uuidV4()
+    let email = "rbalajis25@gmail.com"
+    let args = [
+        "-g", "https://github.com/sch00lb0y/tempCheck.git",
```

Comment:
This test is dependent on your email and your repository. The day you delete your repository, this test will fail.
Please write tests that are not dependent of external factors.